### ISSUE TITLE
[Question]: What would be impacted if we turned off the need for privileged containers

### ISSUE THREAD

**Issue by chadsly** [dockur/windows#491]
### Is your question not already answered in the FAQ?

- [X] I made sure the question is not listed in the [FAQ](https://github.com/dockur/windows/blob/master/readme.md).

### Is this a general question and not a technical issue?

- [X] I am sure my question is not about a technical issue.

### Question

There are some users of this container that are trying to run in kubernetes. One of the requirements of this project is to run "privilege: true".  However, I'd like to explore ways to remove this requirement for improved security and isolation. What approaches or adjustments can be made so that the container operates effectively within Kubernetes without needing or relying on host system access privileges? Any guidance would be greatly appreciated.

**Comment by chadsly** [comment]
There a few other issues that are already closed that were solved by adding the privilege access.  They've shown issues with "NET_ADMIN".  I'm assuming that it is because of something that you're inheriting rather than an issue that you are specifically creating.
https://github.com/dockur/windows/issues/373
https://github.com/dockur/windows/issues/341

**Comment by chadsly** [comment]
That's awesome. I'll try DHCP mode.

**Comment by chadsly** [comment]
I'm able to get "started" in k8s, but I'm having some trouble with some settings.
![image](https://github.com/dockur/windows/assets/9471348/0c473a95-995d-4af9-9c70-194d05792224)

As you can see in the attached image, the container is expecting /dev/net/tun (whatever that is) and NET_ADMIN

Any ideas how to turn on /dev/net/tun without NET_ADMIN?

**Comment by chadsly** [comment]
Update... I was able to add DHCP.  I had to look at the original QEMU image, then their entry.sh script, which led me to the netowrk.sh script.  It's looking for the environment variable to be set.
env:
  - name: DHCP
    value: "Y"

I now have these resultant errors.  We're getting closer, @kroese This is going to be even more awesomer when we solve this.

![image](https://github.com/dockur/windows/assets/9471348/a815a45d-5355-43b7-8536-f786bf3c760a)
